.github/workflows/tests.yml

```diff
@@ -369,7 +371,7 @@ jobs:
 
       - uses: azure/setup-kubectl@v1
         with:
-          version: 'latest'
+          version: 'v1.25.4'
```

Comment:
I mostly did this to have a predictable kubectl version. Not entirely necessary.

Reply:
Yes, that could help to avoid issues but we'll need to think to upgrade it when supporting new Kubernetes versions

Reply:
I will restructure this in a follow up PR, so we use the same kubectl version everywhere in CI

Reply:
I added a follow up issue here: https://github.com/goharbor/harbor-operator/issues/1022